sdk/cosmos/azure-cosmos/azure/cosmos/aio/_container.py

```diff
@@ -320,6 +320,8 @@ def query_items(
         :keyword dict[str, str] initial_headers: Initial headers to be sent as part of the request.
         :keyword response_hook: A callable invoked with the response metadata.
         :paramtype response_hook: Callable[[Dict[str, str], AsyncItemPaged[Dict[str, Any]]], None]
+        :keyword int response_Continuation_Token_Limit_In_Kb: is used to limit the length of continuation token
```

Comment:
let's use pythonic code here and have only lower case with underscores as opposed to camel case